airflow/providers/qubole/example_dags/example_qubole.py

```diff
@@ -55,7 +55,7 @@
         """
     )
 
-    def compare_result_fn(**kwargs):
+    def compare_result_fn():
```

Comment:
Instead of `get_current_context()` (which is something of an escape hatch) we should do this:

```suggestion
    def compare_result_fn(ti):
```

Reply:
Updated to pass in the task instance object and remove the get_current_context() function call.